lib/pusher/client.rb

```diff
@@ -88,6 +105,12 @@ def encrypted?
       @scheme == 'https'
     end
 
+    def cluster=(cluster)
```

Comment:
Calling both is an error. Do we
 - raise?
 - define a clear precedence of one over the other?
 - just go with the last one to be called?

That's pretty much in my descending order of preference. Open to argument, obviously...